linkml/generators/pythongen.py

```diff
@@ -127,8 +138,8 @@ def gen_schema(self) -> str:
         handlerimport = (
             "from linkml_runtime.utils.enumerations import EnumDefinitionImpl"
         )
-        split_descripton = "\n#              ".join(
-            split_line(be(self.schema.description), split_len=100)
+        split_description = "\n#   ".join(
+            d for d in self.schema.description.split("\n")
```

Comment:
the existing code messed up the formatting for the description field so I've just indented it and left the formatting as-is